### DISCUSSION TITLE
4.0.6 not on crates.io yet

### DISCUSSION THREAD

**Discussion by djc** [async-graphql/async-graphql#1000] [Q&A]
I see a GitHub vulnerability referring to a potential DoS and commits updating the version to 4.0.6 but async-graphql 4.0.6 does not appear on crates.io. What's going on with that? When can we expect it to be available on crates.io?

https://github.com/async-graphql/async-graphql/security/advisories/GHSA-xq3c-8gqm-v648

**Comment by sunli829** [comment]
I forgot, it's published now. 😂
